project/projectReadRouter.js

```diff
@@ -21,7 +21,7 @@ router.route("/:id/manifest").get(auth0Middleware(), async (req, res) => {
     if (!await new Project(id).checkUserAccess(user._id, ACTIONS.UPDATE, SCOPES.ALL, ENTITIES.PROJECT)) {
       return respondWithError(res, 403, "You do not have permission to export this project")
     }
-    const manifest = await ProjectFactory.exportManifest(id)
+    const manifest = await ProjectFactory.exportManifest(id, user._id)
```

Comment:
This is reading a Project, we should already have a creator on the Project items. We woudln't want to replace any of them with the person who requested the manifest be created.